components/mdx/ModalImage.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react'
+import Modal from 'react-bootstrap/Modal'
+import Image, { ImageProps } from 'next/image'
+
+const ModalImage: React.FC<ImageProps> = (props: ImageProps) => {
+  const [show, setShow] = React.useState(false)
```

Comment:
Just to keep the same standard between all React code.

```suggestion
import React, { useState } from 'react'
import Modal from 'react-bootstrap/Modal'
import Image, { ImageProps } from 'next/image'

const ModalImage: React.FC<ImageProps> = (props: ImageProps) => {
  const [show, setShow] = useState(false)
```